includes/wikia/services/HubService.class.php

```diff
@@ -61,6 +61,41 @@ public static function getCurrentWikiaVerticalName() {
 	}
 
 	/**
+	 * Get canonical vertical name for given cityId.
+	 * For Lifestyle and Gaming return their names.
+	 * For rest of values return Entertainment.
+	 * @param $cityId
+	 * @return Boolean|String
+	 */
+	public static function getVerticalNameForComscore( $cityId ) {
+		$verticalId = WikiFactoryHub::getInstance()->getVerticalId( $cityId );
+
+		switch ($verticalId) {
+			case WikiFactoryHub::HUB_ID_VIDEO_GAMES:
+				return 'gaming';
+			case WikiFactoryHub::HUB_ID_LIFESTYLE:
+				return 'lifestyle';
+			case WikiFactoryHub::HUB_ID_OTHER:
+				if ( self::isWikiCorporate( $cityId ) ) {
+					return false;
+				} else {
+					return 'lifestyle';
+				}
+			default:
+				return 'entertainment';
+		}
+	}
+
+	private static function isWikiCorporate( $cityId ) {
```

Comment:
If there is no param needed they you could use this one: https://github.com/Wikia/app/blob/dev/includes/wikia/WikiaPageType.class.php#L255

BTW
```getVerticalNameForComscore``` won't be probably used in other places than Comscore, that's why I suggest moving it there. Logic is related to comscore only, strings are hardcoded and not standard ones. What do you think about it?